### PR TITLE
add task

### PR DESCRIPTION
**Pull request by NickZabolotskiy**
c++

### CODE REVIEW

File: Zabolotskiy/Project2/Project2/Debug/Project2.log [hunk 1]
```diff
@@ -0,0 +1,4 @@
+﻿  Source.cpp
+c:\users\никита\desktop\project2\project2\source.cpp(23): warning C4996: 'mbstowcs': This function or variable may be unsafe. Consider using mbstowcs_s instead. To disable deprecation, use _CRT_SECURE_NO_WARNINGS. See online help for details.
```
Comment (Andrew414) [inline Zabolotskiy/Project2/Project2/Debug/Project2.log:2]: Well, this function may indeed be unsafe, you indeed should consider `mbstowcs_s` instead

**Review by Andrew414 — APPROVED** [review]
Same 5/10 because the task is the same as the first one.
I also start thinking about dividing the score by 2 :).

**Comment by NickZabolotskiy** [comment]
We work with Kristy together, together ask you about your problem and actually don't know what can I change in my program. Its very simple basic code:

Inf loop
System process snapshot
Normalizarion input data
Close handle of found process

**Comment by NickZabolotskiy** [comment]
I hope, you don't sink that I can't make this)

**Pull request merged**